### DISCUSSION TITLE
Design System

### DISCUSSION THREAD

**Discussion by lishaduck** [PSDTools/app#41] [General]
Dropping these here so I don't forget. I'll come back and make notes later.

- <https://pub.dev/packages/design_system_annotations>
- <https://www.supernova.io/blog/what-is-a-design-system>
- <https://medium.com/swlh/the-comprehensive-8pt-grid-guide-aa16ff402179>

**Comment by ParkerH27** [comment]
Completely unrelated but i didnt want to make a thing just to ask.... Where is the assets folder where I can put images @lishaduck

**Reply by lishaduck** [reply]
It's `assets/`. Replace `.gitkeep` with the file you want, and add the path of the file to the assets section in the `pubspec.yaml`. Then, run `dart run build_runner build` to trigger FlutterGen which will generate the code to grab the path of the file. Yeah. I know it's a lot, but hey! Think how much **harder** it _could_ be.

**Reply by lishaduck** [reply]
Did you use the [FlutterGen](https://pub.dev/packages/flutter_gen#assets) paths?

**Comment by ParkerH27** [comment]
alr fixed it

**Comment by ParkerH27** [comment]
<img width="760" alt="Screen Shot 2023-05-23 at 9 44 00 AM" src="https://github.com/PSDTools/app/assets/91337218/68c54d8a-d804-4168-9dee-a6e771550a8d">

**Reply by lishaduck** [reply]
Can you make a draft PR so Netlify will create a preview, so I can see it?

**Comment by lishaduck** [comment]
Getting around to writing notes:

Many things should be centralized and standardized.
- The constraints and shadows of `Box`es
- The `Colors`
- The `EdgeInsets`
- The `Icons`
- The `Radius`es
- The `TextStyle`
- The `ThemeData`

**Reply by lishaduck** [reply]
Use widths/heights based on the 8pt[^1] system.

[^1]: 8 pixels.
